nemo_skills/evaluation/evaluator/audio.py

```
@@ -34,13 +35,137 @@ class AudioEvaluatorConfig(BaseEvaluatorConfig):
+
+    def _normalize_numbers(self, text, lang):
+        import num2words
+
+        # Join space-separated thousand groups (e.g. "10 000" -> "10000")
+        text = re.sub(r"(\d)\s+(\d{3})\b", r"\1\2", text)
+
+        # Convert remaining digit sequences to words
+        def _replace(m):
+            try:
+                return num2words.num2words(int(m.group()), lang=lang)
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)